### PR TITLE
Prompt user to reenter credentials when create napps upload fails

### PR DESCRIPTION
**Pull request by Niehaus**
Working on #292 

### :bookmark_tabs: Description of the Change

Added a clear message for user rerun `kytos napps upload` when them has
an invalid or expired token under the status code 401.

### :page_facing_up: Release Notes

- Added clear response for user handle the NApps upload fails.

### CODE REVIEW

File: tests/unit/test_decorators.py [hunk 1]
```diff
@@ -52,6 +52,9 @@ def test_authenticate(self, *args):
         (_, mock_save_token, mock_requests_get, _) = args
         mock_requests_get.return_value = self._expected_response(401)
 
-        self.kytos_auth.authenticate()
+        try:
+            self.kytos_auth.authenticate()
+        except OSError:
+            print("Handle unnecessary OSError")
```
Comment (hdiogenes) [inline tests/unit/test_decorators.py:58]: We need a message that is more useful to the kytos user here.

Comment (Niehaus) [reply tests/unit/test_decorators.py:58]: Which message do you think is more useful? I could think about `Error finding endpoint config request.`

Comment (hdiogenes) [reply tests/unit/test_decorators.py:58]: I think we need to show a message that:
1. Shows the root cause for the error.
2. Tells the user what she can do about it.

Comment (josemauro) [reply tests/unit/test_decorators.py:58]: In fact, this test is evaluating a result or behavior. A new test is being proposed in https://github.com/kytos/kytos-utils/pull/317

Comment (josemauro) [inline tests/unit/test_decorators.py:58]: This PR are being closed once https://github.com/kytos/kytos-utils/pull/317 fixes the problems existent here.

**Pull request closed without merging**